test/test_epoll.cpp

```diff
@@ -34,7 +35,6 @@ TEST(CEPoll, InfiniteWait)
     EXPECT_EQ(srt_cleanup(), 0);
```

Comment:
Although I wrote it, now I think try-catch is not required here.
`srt_epoll_uwat()` can be added here as well.
```
SRT_EPOLL_EVENT events[2];
ASSERT_EQ(srt_epoll_uwait(epoll_id, events, 2, -1), SRT_ERROR);
```

Reply:
Well, indeed it wasn't - `srt_epoll_release` like any other API function is not expected to throw exceptions.